docs/rst/statistics_backend/get_data.rst

```diff
@@ -2,20 +2,22 @@
 
 .. _statistics_backend_get_data:
 
-Retrieve statistical data
--------------------------
+Get statistical data
+--------------------
 
 .. warning::
     This feature is currently not supported.
     It will be implemented on a future release of *Fast DDS Statistics Backend*.
 
 *Fast DDS Statistics Backend* provides two overloads of |get_data-api| to retrieve statistical data of a given
-|DataKind-api| within a time frame.
+|DataKind-api| within a time frame (for more information about all the reported |DataKind-api|, please refer to
+:ref:`types_data_kind`).
 This time interval is evenly divided into the specified number of bins, each one with size
 :math:`(t_to - t_from)/(# of bins)`.
 For each of these bins, a new |StatisticsData-api| value is calculated applying the given |StatisticKind-api| to all the
 data points in it.
 The result is a collection of |StatisticsData-api| elements with size equal to the number of specified bins.
+If the number of bins is set to zero, then all data points are returned and no statistic is calculated for the series.
```

Comment:
NIT: I think it is an important feature. I would suggest to explain it wider, at least add a break line.